src/Pact/Types/Term.hs

```diff
@@ -198,6 +200,9 @@ instance ToJSON KeySet where
     toJSON (KeySet k f) = object ["keys" .= k, "pred" .= f]
 
 
+keysetFromList :: [PublicKey] -> Name -> KeySet
```

Comment:
OK last request:
```suggestion
mkKeyset :: [PublicKey] -> Text -> KeySet
```
as Name $ BareName is the only sane default

Reply:
Addressed